explorer/templates.go

```diff
@@ -259,5 +268,18 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"theme": func() string {
 			return netTheme
 		},
+		"emptyArray": func(length int) []int {
+			if length < 0 {
+				length = 0
+			}
+			return make([]int, length)
+		},
+		"first30Txs": func(txs []TrimmedTxInfo) []TrimmedTxInfo {
```

Comment:
I'd prefer to generalize this to something like "clipSlice" and give it a second param, `N`.